### PR TITLE
Zoom plus center params and fix initial embed code

### LINKED ISSUE #71: Embed code does not reflect initial map layers
Example: Load `https://wearefairphone.github.io/fprsmap/?show=angels,events,shops` in your browser.

The embed popup will only insert the default map layers in the code, which are at the moment `angels,events`.

### PR DESCRIPTION
**Pull request by Ingo-FP-Angel**
Fixes #71 and implements #45

### CODE REVIEW

**Comment by StefanBrand** [comment]
Wow, thanks for your acitivity lately! Makes me want to jump in too. :))

**Comment by Ingo-FP-Angel** [comment]
Yeah, `CURRENTDATE` was marked as unused in my IDE so I applied the - in German - _Pfadfinderregel_ to leave the place more clean than I found it.
The agenda.json indeed does not contain past events.

I'll look into the `getQueries()` remark.

**Comment by Ingo-FP-Angel** [comment]
I forgot to extend the Readme to describe the new query params.
Will do it tomorrow.

**Comment by StefanBrand** [comment]
Ah, good one! 😅
-- 
Diese Nachricht wurde von meinem Fairphone mit K-9 Mail gesendet.

Am 18. September 2019 18:59:02 MESZ schrieb Ingo-FP-Angel <notifications@github.com>:
>I forgot to extend the Readme to describe the new query params.
>Will do it tomorrow.

**Comment by Ingo-FP-Angel** [comment]
Together with the Readme updated I also fixed a bug I introduced (in case no layers were given as query parameter initially).

File: README.md [hunk 1]
```diff
@@ -30,8 +30,16 @@ The categories mentioned in the previous paragraph can be used to define an init
 - Say you want to promote Fairphone Community events. Append `?show=events` to the URI and only meetups and events will be loaded into the map at start.
 - Multiple parameters are also possible: `https://map.fairphone.community/?show=events,angels` will display both the `events` and the `angels` layers.
 
+To further customize the initial state of the map, the following parameters are also possible:
+
+- If you want to center the map on a certain location use `center=<lat>,<lng>` where `<lat>` is the latitude and `<lng>` the longitude (defaults to `center=49.8158683,6.129675`).
+- In order to zoom into the map, use `zoom=<value>` where `<value>` is an integral number between 2 and 18 (defaults to `zoom=2`).
```
Comment (StefanBrand) [inline README.md]: `integral` should be `integer` I think :)

**Comment by Ingo-FP-Angel** [comment]
> At some point we could look into filling the address bar with the current query parameters too.

Very nice idea! Was easy to implement, too ;-)

File: README.md (hunk 1 above)
Comment (Ingo-FP-Angel) [reply README.md]: Done

**Comment by Ingo-FP-Angel** [comment]
The way I implemented it is to replace the current history, i.e. you'd be back to DuckDuckGo after one click.

**Comment by StefanBrand** [comment]
Wow, very cool! :D Great, thank you!!!

**Pull request merged**